trello_api.py

```diff
@@ -2,20 +2,73 @@
 import os
 from copy import deepcopy
 from to_do_item import ToDoItem
+from to_do_item import Status
+from dotenv import find_dotenv, load_dotenv
 
 API_KEY = os.getenv('API_KEY')
 TOKEN = os.getenv('TOKEN')
 BOARD_ID = os.getenv('BOARD_ID')
-TO_DO_LIST_ID = os.getenv('TO_DO_LIST_ID')
-DONE_LIST_ID = os.getenv('DONE_LIST_ID')
 
 CREDENTIALS = { 'key': API_KEY, 'token': TOKEN }
 
 TRELLO_API_BASE_URL = 'https://api.trello.com/1'
 
+TO_DO_LIST_ID = None
+DOING_LIST_ID = None
+DONE_LIST_ID = None
+
+def reload_credentials():
+    global CREDENTIALS
+    CREDENTIALS = { 'key': API_KEY, 'token': TOKEN }
+
+def load_api_key_if_none():
+    global API_KEY
+    if API_KEY is None:
+        file_path = find_dotenv('.env')
+        load_dotenv(file_path, override=True)
+        API_KEY = os.getenv('API_KEY')
+        reload_credentials()
+
+def load_token_if_none():
+    global TOKEN
+    if TOKEN is None:
+        file_path = find_dotenv('.env')
+        load_dotenv(file_path, override=True)
+        TOKEN = os.getenv('TOKEN')
+        reload_credentials()
+
+def load_board_id_if_none():
+    global BOARD_ID
+    if BOARD_ID is None:
+        BOARD_ID = os.getenv('BOARD_ID')
+
+def load_to_do_list_id_if_none():
+    load_board_id_if_none()
+    global TO_DO_LIST_ID
+    if TO_DO_LIST_ID is None:
+        TO_DO_LIST_ID = get_list_id_by_name("To Do", BOARD_ID)
```

Comment:
Hardcoded strings. This implicit requirement should be documented in the `README.md` file